app/helpers/application_helper.rb

```diff
@@ -1,3 +1,10 @@
 module ApplicationHelper
-  def full_title; end
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
```

Comment:
dùng toán tử 3 ngôi cho gọn code nếu ko quá dài em nhé
page_title.empty? ? base_title : "#{page_title} | #{base_title}"